File: app/src/main/java/com/d4rk/androidtutorials/java/ui/screens/android/AndroidStudioFragment.java
package com.d4rk.androidtutorials.java.ui.screens.android;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import com.d4rk.androidtutorials.java.R;

public class AndroidStudioFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.preferences_android_studio, rootKey);
    }
}

File: app/src/main/kotlin/com/d4rk/androidtutorials/java/ui/android/AndroidStudioFragment.kt
package com.d4rk.androidtutorials.java.ui.android
import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.d4rk.androidtutorials.java.R
class AndroidStudioFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.preferences_android_studio, rootKey)
    }
}